tests/helloidris2/test001/run

```diff
@@ -1,3 +1,3 @@
-idris2 --no-banner --no-color --console-width 0 -p helloidris2 Main.idr < input
+$1 --no-banner --no-color --console-width 0 -p helloidris2 Main.idr < input
```

Comment:
Did not know `idris2` was passed as `$1` for these `run` files, good to know!

Reply:
I guess it's only relevant for the main project or people developing a separate backend who
want to run their tests on the modified `idris2` without having to install it first in case
it's broken.